packages/commons-test/src/eventStoreTestUtils.ts

```diff
@@ -22,6 +22,7 @@ export const eventStoreSchema = {
   attribute: "attribute",
   catalog: "catalog",
   tenant: "tenant",
+  purpose: "purpose",
 } as const;
 
 export const EventStoreSchema = z.enum([
```

Comment:
@taglioni-r In #377 I improved these `eventStoreTestUtils.ts` methods, to avoid wrong usages (as it happened in that PR). This model is now in `models.ts` and already contains purpose - after you rebase, you need to update the methods to support Purpose events, and adapt the `writePurposeInEventstore` and the corresponding read in purpose test utils accordingly

Reply:
Updated, thanks